controllers/noobaa-rules.yaml

```diff
@@ -59,16 +61,14 @@
            system_type: NooBaa
            system_vendor: Red Hat
            origin: data-federation-service
-         record: noobaa_system_health_status
+         record: namespace:noobaa_system_health_status
    - name: bucket-state-alert.rules
      rules:
-       - alert: BucketPolicyErrorState
+       - alert: NooBaaBucketPolicyErrorState
```

Comment:
I don't see a `for` duration for this alert. Is it on purpose?

Reply:
It's here: https://github.com/red-hat-storage/mcg-osd-deployer/pull/67/files#diff-c2d8ea863a8b41a65ff336525a37d8a9b198d683b1dae92c7d4be6f06c446d7eR74.

Reply:
ack